compiler/modules/front.syntax/parseExpr.bal

```diff
@@ -145,7 +145,8 @@ function finishTypeTestExpr(Tokenizer tok, Expr expr, boolean negated, Position
 function parseRangeExpr(Tokenizer tok) returns RangeExpr|err:Syntax {
     Position startPos = tok.currentStartPos();
     Expr lower = check parseAdditiveExpr(tok);
-    Position opPos = check tok.expectEnd("..<");
+    Position opPos = tok.currentStartPos();
```

Comment:
I suggest you add an `expectStart` method to deal with these.

Reply:
fixed